### PR TITLE
Mainnet unfinalized blocks duration

### PR DESCRIPTION
**Pull request by fboucquez**
Updated:

**testnet:**
votingUnfinalizedBlocksDuration: 0m
nonVotingUnfinalizedBlocksDuration: 8m

**mainnet:**
votingUnfinalizedBlocksDuration: 0m
nonVotingUnfinalizedBlocksDuration: 10m

### CODE REVIEW

File: presets/mainnet/network.yml [hunk 1]
```diff
@@ -50,6 +50,7 @@ fileDatabaseBatchSize: 100
 minProofSize: 0
 syncTimeout: 5m
 maxChildNamespaces: 100
+unfinalizedBlocksDuration: 10m
```
Comment (Wayonb) [inline presets/mainnet/network.yml:53]: Is this only for non voting nodes?

**Review by Wayonb — CHANGES_REQUESTED** [review]

File: presets/mainnet/network.yml (hunk 1 above)
Comment (fboucquez) [reply presets/mainnet/network.yml]: Added custom value for voting nodes

File: presets/mainnet/network.yml [hunk 2]
```diff
@@ -50,6 +50,8 @@ fileDatabaseBatchSize: 100
 minProofSize: 0
 syncTimeout: 5m
 maxChildNamespaces: 100
+unfinalizedBlocksDuration: 0m
+votingUnfinalizedBlocksDuration: 10m
```
Comment (Wayonb) [inline presets/mainnet/network.yml:54]: should probably be votingUnfinalizedBlocksDuration => nonVotingUnfinalizedBlocksDuration?

File: src/service/ConfigService.ts [hunk 3]
```diff
@@ -311,6 +311,9 @@ export class ConfigService {
             friendlyName: nodePreset?.friendlyName || account.friendlyName,
             harvesterSigningPrivateKey: harvesterSigningPrivateKey,
             harvesterVrfPrivateKey: harvesterVrfPrivateKey,
+            unfinalizedBlocksDuration: nodePreset.voting
```
Comment (Wayonb) [inline src/service/ConfigService.ts:314]: should be the reverse?

**Review by Wayonb — CHANGES_REQUESTED** [review]

File: src/service/ConfigService.ts (hunk 3 above)
Comment (fboucquez) [reply src/service/ConfigService.ts:314]: it was fine, now it's clearer

File: presets/shared.yml [hunk 4]
```diff
@@ -99,7 +99,8 @@ finalizationSize: 10000
 finalizationThreshold: 7000
 maxHashesPerPoint: 256
 prevoteBlocksMultiple: 4
-unfinalizedBlocksDuration: 0m
+nonVotingUnfinalizedBlocksDuration: 0m
+votingUnfinalizedBlocksDuration: 8m
```
Comment (Wayonb) [inline presets/shared.yml:103]: When the node is voting - votingUnfinalizedBlocksDuration: 0m
Nonvoting node - nonVotingUnfinalizedBlocksDuration: 8m

File: presets/mainnet/network.yml [hunk 5]
```diff
@@ -50,6 +50,8 @@ fileDatabaseBatchSize: 100
 minProofSize: 0
 syncTimeout: 5m
 maxChildNamespaces: 100
+nonVotingUnfinalizedBlocksDuration: 0m
+votingUnfinalizedBlocksDuration: 10m
```
Comment (Wayonb) [inline presets/mainnet/network.yml:54]: When the node is voting - votingUnfinalizedBlocksDuration: 0m
Nonvoting node - nonVotingUnfinalizedBlocksDuration: 10m

**Review by Wayonb — CHANGES_REQUESTED** [review]

**Review by rg911 — APPROVED** [review]

**Review by Wayonb — APPROVED** [review]

**Pull request merged**